source/blog/2017-12-08-ovirt-ansible-roles-split.html.md

```diff
@@ -0,0 +1,104 @@
+---
+title: oVirt roles Ansible Galaxy integration
+author: omachace
+tags: Ansible, Roles, Automation, Configuration management, Galaxy
+date: 2017-12-08 15:00:00 CET
+comments: true
+published: true
+---
+
+In 4.2 release we have splitted our oVirt Ansible roles into separate RPM packages and also separate git repositories, so it is possible for user to install specific role either from Ansible Galaxy or as RPM.
+
+READMORE
+
+## The reason
+The reason to split the roles into separate packages and git repositories was mainly the usage from the Ansible Tower. Since Ansible Galaxy is only integrated with github you need to store your Ansible role in single git repostiory in order to have separate Ansible role in Galaxy. Previously we used one single repository where we have stored all the roles, but that made it unusable from Ansible Tower. So as you can see on image below we now have many roles in Ansible Galaxy under oVirt user:
+
+![oVirt roles in Ansible Galaxy](galaxy_roles.png)
+
+## How to install the roles 
+There are still two posibilities of installing the roles, you can do it either via Ansible Galaxy or from oVirt repository.
+
+### Ansible Galaxy
+Newly you can now install just a single role, instead of downloading them all at once.
```

Comment:
You are now able to install just a single role and not necessarily all of them at once like in previous versions

Reply:
done